hs-abci-examples/nameservice/README.md

```diff
@@ -1,23 +1,147 @@
 # nameservice
 
-## Metrics
+## Metrics via Prometheus
```

Comment:
Before commenting on any of this, is there any way to have kibana pull metrics from prometheus in the same way datadog does. This would be good to fix some confusing asymmetry between server metrics and application metrics, and would also be easier to explain that you can now use either Kibana or datadog to view anything you want, or both.

Reply:
Good question. I’ll investigate.

Reply:
@blinky3713 

It seems [`MetricBeat`](https://www.elastic.co/products/beats/metricbeat) is an option from the ELK folks that can scrape a prometheus metrics server. Given we're already using ELK stack, this seems to be a pretty good option.

Similar to the datadog agent, it also has a [`docker-compose`](https://github.com/shazChaudhry/docker-elastic/blob/master/metricbeat-docker-compose.yml).